toolkit-impl/src/test/java/com/terracotta/tooolkit/mockl2/ds/ToolkitListTest.java

```diff
@@ -69,7 +69,7 @@ public void logicalInvoke(Object object, LogicalOperation method, Object[] param
         
       }
     });
-    toolkitMap.put("organzation", "softwareag");
+    toolkitMap.put("organization", "IBM");
```

Comment:
`IBM`, or `Super iPaaS Integration LLC, an IBM Company` ?

Reply:
it's just a test, not a copyright